2.3/configuration/the-rules.md

```diff
@@ -0,0 +1,88 @@
+# The Rules
+
+[[toc]]
+
+## Overview
+
+ls-lint provides multiple rules out of the box:
+
+| Rule               | Alias                | Description                                                                            |
+|--------------------|----------------------|----------------------------------------------------------------------------------------| 
+| [regex](#regex)    | -                    | Matches regex pattern: ^{pattern}$                                                     |
+| [exists](#exists)  | -                    | Allow or disallow the existence of _`N`_ or _`N-M`_ files. Also works for directories. |
```

Comment:
Here the N and N-M might be misunderstood by some people. Please look at other tools documentation to see how they refers to such arguments (tail -n ,head -n, grep -C, …)